tests/test_checkout.py

```diff
@@ -628,7 +633,7 @@ def test_create_order_creates_expected_events(
         user=customer_user if not is_anonymous_user else AnonymousUser(),
     )
 
-    # Ensure only two events were created, and retrieve them
+    # Ensure only three events were created, and retrieve them
```

Comment:
If there are three of them, you need to unpack one more. And no need to mock + please check the event has the expected data from the db object.

Reply:
I forgot to fix this test, there's only 2 events because staff_email doesn't create event (we doesn't need it to)